block_test.go

```diff
@@ -35,6 +37,10 @@ func runnerWithRendererParameters(parameters HtmlRendererParameters) func(string
 		htmlFlags := 0
 		htmlFlags |= HTML_USE_XHTML
 
+		if parameters.SanitizedAnchorNameOverride == nil {
+			parameters.SanitizedAnchorNameOverride = sanitized_anchor_name.Create
+		}
+
 		renderer := HtmlRendererWithParameters(htmlFlags, "", "", parameters)
```

Comment:
Are the 4 added lines above needed? `HtmlRendererWithParameters` already sets the default value for `SanitizedAnchorNameOverride` if it's `nil`.